src/SFA.DAS.Roatp.CourseManagement.UnitTests/Application/Standards/Commands/UpdateContactDetailsCommandHandlerTests.cs

```diff
@@ -22,13 +24,28 @@ public class UpdateContactDetailsCommandHandlerTests
             UpdateContactDetailsCommandHandler sut,
             UpdateContactDetailsCommand command,
             CancellationToken cancellationToken,
-            UpdateContactDetailsRequest request)
+            UpdateProviderCourseRequest request,
+            GetProviderCourseResponse apiResponse)
         {
-            apiClientMock.Setup(a => a.PostWithResponseCode<UpdateContactDetailsRequest>(It.IsAny<UpdateContactDetailsRequest>())).ReturnsAsync(new ApiResponse<UpdateContactDetailsRequest>(request, HttpStatusCode.NoContent, string.Empty));
+            apiClientMock.Setup(a => a.GetWithResponseCode<GetProviderCourseResponse>(It.IsAny<GetProviderCourseRequest>())).ReturnsAsync(new ApiResponse<GetProviderCourseResponse>(apiResponse, HttpStatusCode.OK, string.Empty));
+
+            apiClientMock.Setup(a => a.PostWithResponseCode<UpdateProviderCourseRequest>(It.IsAny<UpdateProviderCourseRequest>())).ReturnsAsync(new ApiResponse<UpdateProviderCourseRequest>(request, HttpStatusCode.NoContent, string.Empty));
 
             await sut.Handle(command, cancellationToken);
 
-            apiClientMock.Verify(a => a.PostWithResponseCode<UpdateContactDetailsRequest>(It.IsAny<UpdateContactDetailsRequest>()), Times.Once);
+            apiClientMock.Verify(a => a.PostWithResponseCode<UpdateProviderCourseRequest>(It.IsAny<UpdateProviderCourseRequest>()), Times.Once);
+        }
```

Comment:
Tests missing for conversion from Command to Request model. Same set of tests should be added to other handler as well.